packages/table/src/HeaderRow.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Checkbox from '@leafygreen-ui/checkbox';
+import { uiColors } from '@leafygreen-ui/palette';
+import { css } from '@leafygreen-ui/emotion';
+import { useTableContext, Types } from './TableContext';
+
+const thStyles = css`
+  width: 40px;
+  border-width: 0 1px 3px 1px;
+  border-color: ${uiColors.gray.light2};
+  border-style: solid;
+`;
+
+export interface HeaderRowProps extends React.ComponentPropsWithoutRef<'tr'> {
+  indeterminate?: boolean;
```

Comment:
This doesn't seem like it should be a public prop. Can this be pulled from the table context instead?

Reply:
Yes good call I actually don't think either of these are required.